### PR TITLE
Content: news about design system + UI kit Italia + Bootstrap Italia

### PR DESCRIPTION
**Pull request by gianvitofanelli**
# News
- [x] Text
- [x] Review

# Cards
- [x] Thumbnail graphic by @Fupete 
- [x] Add cards to home
- [x] Add cards to Community

### CODE REVIEW

File: src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# ————————————————————————————————————————————
+# CREATEPAGE GATSBY
+# ————————————————————————————————————————————
+metadata:
+  template:
+    name: level3 # article?
+  archive: notizie
+seo:
+  name: "Nuovi rilasci del design system del Paese e delle risorse di progettazione e sviluppo"
+  description: "Tra le principali novità, il player video attento a privacy e accessibilità, e nuove schede informative."
```
Comment (danielaiozzo) [inline src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml:10]:
```suggestion
  description: "Tra le principali novità, il player video attento a privacy e accessibilità e nuove schede informative."
```

File: src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          color: "secondary"
+        label: Notizie
+        url: "../"
+
+components:
+  hero:
+    background: null
+    centered: true
+    title: "Nuovi rilasci del design system del Paese e delle risorse di progettazione e sviluppo"
+    subtitle: "Tra le principali novità, il player video attento a privacy e accessibilità, e nuove schede informative"
```
Comment (danielaiozzo) [inline src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml:52]:
```suggestion
    subtitle: "Tra le principali novità, il player video attento a privacy e accessibilità e nuove schede informative"
```

File: src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+      id: corpoNotizia
+      components:
+        - name: TextImageCta
+          # title: Titoletto interno
+          text: |
+           Sono appena stati rilasciati molti aggiornamenti per la documentazione del **design system del Paese v1.0.0-alpha.12** ([changelog](/design-system/fondamenti/versionamento#changelog-documentazione)), **UI Kit Italia v3.2.0** ([changelog](https://github.com/italia/design-ui-kit/releases)) e **Bootstrap Italia v2.7.4** ([changelog](https://github.com/italia/bootstrap-italia/releases)). 
+
+            Tra le principali novità di questi aggiornamenti:  
+
+            - nuovo componente **["Video player"](/design-system/componenti/video-player/)** attento a privacy e accessibilità. Nelle prossime settimane, Designers Italia sarà il primo terreno di sperimentazione con la pubblicazione dell’archivio media, del nuovo meccanismo di gestione cookie di piattaforme di terze parti e l’aggiornamento del kit privacy;
```
Comment (danielaiozzo) [inline src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml:111]:
```suggestion
            - nuovo componente **["Video player"](/design-system/componenti/video-player/)** attento a privacy e accessibilità. Nelle prossime settimane, Designers Italia sarà il primo terreno di sperimentazione del nuovo meccanismo di gestione cookie di piattaforme di terze parti con la pubblicazione dell’archivio media e l’aggiornamento del kit privacy; 
```

File: src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+            Tra le principali novità di questi aggiornamenti:  
+
+            - nuovo componente **["Video player"](/design-system/componenti/video-player/)** attento a privacy e accessibilità. Nelle prossime settimane, Designers Italia sarà il primo terreno di sperimentazione con la pubblicazione dell’archivio media, del nuovo meccanismo di gestione cookie di piattaforme di terze parti e l’aggiornamento del kit privacy; 
+            - la stesura della **[documentazione](/design-system/come-iniziare/)** procede spedita verso la beta prevista in Q4 2023 con nuove schede "Fondamenti" e aggiornamenti di molte schede componenti con istruzioni per l’uso e specifiche di design;  
+            - nuovi **componenti “Video player” e “Hero”** in UI Kit Italia; 
+            - aggiornamento consistente di tutta la **tipografia** di **Bootstrap Italia** per coerenza con UI Kit Italia;  
+            - molti miglioramenti e correzioni bug sia su UI Kit Italia che Bootstrap Italia con contributi anche dalla community (grazie a @Virtute90, @PiemP, @Simososi).
```
Comment (danielaiozzo) [inline src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml:115]:
```suggestion
            - numerosi miglioramenti e correzioni bug sia su UI Kit Italia che Bootstrap Italia con contributi anche dalla community (grazie a @Virtute90, @PiemP, @Simososi). 
```

File: src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    - menu: false
+      centered: true
+      # title: Stiamo costruendo la nuova casa del progetto per supportare la trasformazione digitale della Pubblica Amministrazione
+      # text:
+      id: corpoNotizia
+      components:
+        - name: TextImageCta
+          # title: Titoletto interno
+          text: |
+           Sono appena stati rilasciati molti aggiornamenti per la documentazione del **design system del Paese v1.0.0-alpha.12** ([changelog](/design-system/fondamenti/versionamento#changelog-documentazione)), **UI Kit Italia v3.2.0** ([changelog](https://github.com/italia/design-ui-kit/releases)) e **Bootstrap Italia v2.7.4** ([changelog](https://github.com/italia/bootstrap-italia/releases)).
```
Comment (danielaiozzo) [inline src/data/content/community/notizie/20231024-nuovi-rilasci-del-design-system-del-paese-e-delle-risorse-di-progettazione-e-sviluppo.yaml:107]:
```suggestion
           Sono appena stati rilasciati vari aggiornamenti per la documentazione del **design system del Paese v1.0.0-alpha.12** ([changelog](/design-system/fondamenti/versionamento#changelog-documentazione)), **UI Kit Italia v3.2.0** ([changelog](https://github.com/italia/design-ui-kit/releases)) e **Bootstrap Italia v2.7.4** ([changelog](https://github.com/italia/bootstrap-italia/releases)). 
```

File: src/data/content/index.yaml [hunk 6]
```diff
@@ -235,48 +235,34 @@ components:
     #   tag:
     #   label: Community Lab
 
-    - title: 'Aziende sanitarie locali: un nuovo modello per migliorare il sito web'
+    - title: 'Nuovi rilasci del design system del Paese e delle risorse di progettazione e sviluppo'
       headingLevel: 3
       customCol: col-lg-4
-      text: Designers Italia avvia la fase di ricerca con gli attori interessati.
-      img: /images/community/20231017-aziende-sanitarie-locali-un-nuovo-modello-per-migliorare-il-sito-web-1400x723.png
+      text: Tra le principali novità, il player video attento a privacy e accessibilità, e nuove schede informative.
```
Comment (danielaiozzo) [inline src/data/content/index.yaml:241]:
```suggestion
      text: Tra le principali novità, il player video attento a privacy e accessibilità e nuove schede informative.
```

**Review by danielaiozzo — CHANGES_REQUESTED** [review]

File: src/data/content/community.yaml [hunk 7]
```diff
@@ -213,6 +117,20 @@ components:
 
       cards:
 
+      - title: 'Nuovi rilasci del design system del Paese e delle risorse di progettazione e sviluppo'
+        headingLevel: 4
+        customCol: col-lg-4
+        text: Tra le principali novità, il player video attento a privacy e accessibilità, e nuove schede informative.
```
Comment (danielaiozzo) [inline src/data/content/community.yaml:123]:
```suggestion
        text: Tra le principali novità, il player video attento a privacy e accessibilità e nuove schede informative.
```

**Review by danielaiozzo — CHANGES_REQUESTED** [review]

**Review by astagi — APPROVED** [review]

**Review by danielaiozzo — APPROVED** [review]

**Review by Fupete — APPROVED** [review]
I will merge it tomorrow morning.

**Pull request merged**